datafusion/physical-expr/src/expressions/literal.rs

```diff
@@ -66,8 +66,7 @@ impl Literal {
         value: ScalarValue,
         metadata: Option<FieldMetadata>,
     ) -> Self {
-        let mut field =
-            Field::new(format!("{value}"), value.data_type(), value.is_null());
+        let mut field = Field::new("lit".to_string(), value.data_type(), value.is_null());
```

Comment:
If we see string allocations on the hot path, perhaps we could do something like cache the FieldRef for each datatype or something 🤔

Reply:
(As a follow on PR, to be clera)